n2_lmbox_lan.py

```diff
@@ -72,9 +81,10 @@ def send_recv(self, message):
             self.logger.fatal("Could not send message %s. Error: %s" % (message.strip(), e))
             ret['retcode'] = -2
             return ret
-        time.sleep(1) # Giving the device more time to respond than normal LAN device
+        time.sleep(2) # Giving the device more time to respond than normal LAN device
         try:
             ret['data'] = self._device.recv(1024)
+            self.logger.debug(f'data: {ret["data"]}')
```

Comment:
I would get rid of this, since it is not really easy to interpret and will swamp the log files

Reply:
Gone in the latest update